starter-code/css/main.css

```diff
@@ -7,4 +7,99 @@ light blue: #1c5380
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
+  background-color: #1c5380;
 }
+
+header {
+  display: flex;
+  justify-content: space-between;
+  background-color: #1c5380;
+}
+
+#brand {
+  width: 50%;
+  height: 50%;
+}
+
+#home {
+  background-color: #1c5380;
+  color: white;
+  border-style: none;
+  color: white;
+  box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2), 0 6px 20px 0 rgba(0,0,0,0.19);
+}
+
+#phone {
+  background-image: url("../images/phones.png");
```

Comment:
Using `background-image` is good, but mostly used when displaying big pictures supposed to take entire sections of the screen, because it allows it to adapt well to various screen sizes.
The problem with this, is that `cover` will sometimes crop parts of your images to avoid distorting them, which is not what you ant here. Thus, simply placing an `<img />` tag in your html and giving it dimensions in css would have been good.